app/routes/repos.py

```
@@ -27,27 +107,83 @@ async def get_repos(
+                        status_code=status.HTTP_400_BAD_REQUEST,
+                    )
+            except Exception as e:
+                return APIResponse.error(
+                    message=f"Failed to fetch repositories: {str(e)}",
+                    status_code=status.HTTP_503_SERVICE_UNAVAILABLE,
+                )
+        return APIResponse.error(
+            message=constants.TOKEN_NOT_FOUND, status_code=status.HTTP_404_NOT_FOUND
+        )
+
+    except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)